pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -180,13 +180,13 @@ func (ac *Config) poolType() string {
 	return ""
 }
 
-// getDisk return the all disks of a certain type(e.g. sparse, disk) which is specified in spc.
-func (ac *Config) getDisk() (*ndmapis.DiskList, error) {
+// getBlockDevice return the all disks of a certain type(e.g. sparse, blockdevice) which is specified in spc.
+func (ac *Config) getBlockDevice() (*ndmapis.BlockDeviceList, error) {
 	diskFilterLabel := diskFilterConstraint(ac.Spc.Spec.Type)
-	dL, err := ac.DiskClient.List(v1.ListOptions{LabelSelector: diskFilterLabel})
+	bdL, err := ac.BlockDeviceClient.List(metav1.ListOptions{LabelSelector: diskFilterLabel})
 	if err != nil {
 		return nil, err
 	}
-	dl := dL.Filter(disk.FilterInactiveReverse)
-	return dl.DiskList, nil
+	bdl := bdL.Filter(blockdevice.FilterClaimedDevices)
```

Comment:
we changed `FilterInactiveReverse` to `FilterClaimedDevices`.. is this fine?

Reply:
Should I add `FilterInactiveReverse` filter? I think If block device is not active can't we claim block device?
cc: @vishnuitta @akhilerm

Reply:
Added `FilterInactiveReverse`